test/functional/page_objects/visualize_chart_page.ts

```diff
@@ -408,7 +431,9 @@ export function VisualizeChartPageProvider({ getService, getPageObjects }: FtrPr
 
         await this.waitForVisualizationRenderingStabilized();
         // arbitrary color chosen, any available would do
-        const arbitraryColor = (await this.isVisTypeXYChart()) ? '#d36086' : '#EF843C';
+        const arbitraryColor = (await this.isNewLibraryChart(xyChartSelector))
```

Comment:
```suggestion
        const arbitraryColor = (await this.isNewLibraryChart(chartSelector))
```

Reply:
I want this condition only for the xy chart